### PR TITLE
Milestone 3: Game logic

### PR DESCRIPTION
**Pull request by carloso0114**
In this milestone we develop a the logics of the game, based on conditions that checks for the player names, moves, and if the player win, loose or if its a draw. Asking to start again if the user wants to keep playing.

### CODE REVIEW

File: lib/methods.rb [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  puts "                 _________________
+                |     |     |     |
+                |  #{new_game.board[0][0]}  |  #{new_game.board[0][1]}  |  #{new_game.board[0][2]}  |
+                |_____|_____|_____|
+                |     |     |     |
+                |  #{new_game.board[1][0]}  |  #{new_game.board[1][1]}  |  #{new_game.board[1][2]}  |
+                |_____|_____|_____|
+                |     |     |     |
+                |  #{new_game.board[2][0]}  |  #{new_game.board[2][1]}  |  #{new_game.board[2][2]}  |
+                |_____|_____|_____|"
```
Comment (elmejdki) [inline lib/methods.rb]: There should be no print or puts outside of main.rb

File: lib/player.rb [hunk 2]
```diff
@@ -0,0 +1,8 @@
+class Player
+  attr_accessor :name, :symbol
```
Comment (elmejdki) [inline lib/player.rb]: change attr_accessor to attr_reader cause you don't need to write the name and symbol you are just reading.

**Review by elmejdki — CHANGES_REQUESTED** [review]
# status: require changes :recycle: 

Great job guys :100:, you have a wonderful job on this project :clap: :clap: :clap: :clap:, you still have only some minor changes :hammer: :hammer_and_pick: :toolbox: to get it approved :checkered_flag:.

Please check the comments below:
:point_down: :point_down: :point_down: :point_down: :point_down:

## Required Changes:

- [x] Remove the .gitkeep file from the lib directory.
- [x] Please add a descriptive Readme to your project.

### Please make all the required changes before you submit another code review.
## Happy Coding :tophat: 
_If you have any questions please don't hesitate to contact me on slack._

**Review by mahmoud717 — APPROVED** [review]
### STATUS: APPROVED ✔️ ✔️ 

**hi @carloso0114  & @Varun2323**

**Great job with making all of the requirements. the project is approved, you can go ahead and merge this pull request**

**Pull request merged**